demo/demo-spring-boot-transport/demo-spring-boot-pojo-client/src/test/java/io/servicecomb/springboot/pojo/client/PojoClientIT.java

```diff
@@ -0,0 +1,24 @@
+package io.servicecomb.springboot.pojo.client;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import io.servicecomb.demo.TestMgr;
+
+public class PojoClientIT {
+
+    @Before
+    public void setUp() throws Exception {
+        TestMgr.errors().clear();
+    }
+
+    @Test
+    public void clientGetsNoError() throws Exception {
+        PojoClient.main(new String[0]);
```

Comment:
please start spring application with the following command to fix the test error
```
SpringApplication.run(PojoClient.class);
```